crates/proc-macro-srv/src/abis/mod.rs

```diff
@@ -68,7 +77,39 @@ impl Abi {
         lib: &Library,
         symbol_name: String,
         info: RustCInfo,
+        #[cfg_attr(not(feature = "sysroot-abi"), allow(unused_variables))] version_string: String,
```

Comment:
Maybe put the version string in `RustCInfo`?

Reply:
Ahh yes I meant to do that to avoid reading twice from the object file!